driver_test.go

```diff
@@ -1022,26 +1044,26 @@ func TestEnableResultCache(t *testing.T) {
 	testEnableResultCachePageSized(t, connDB, vCtx, 0)
 }
 
-//func TestConnectionClosure(t *testing.T) {
-//	adminDB := openConnection(t, "test_connection_closed_pre")
-//	defer closeConnection(t, adminDB, "test_connection_closed_post")
-//	const userQuery = "select 1 as test"
+// func TestConnectionClosure(t *testing.T) {
```

Comment:
Need to undo extra whitespaces for this function.

Reply:
Done